### ISSUE TITLE
[Bug] Double input on MacOS Sonoma with MNK

### ISSUE THREAD

**Issue by fishwasherr** [unknownskl/xbox-xcloud-client#1182]
### Describe the bug

MNK input is doubled, making it really hard to do anything. I.e. Warframe successfully recognises MNK but each input is inputted twice; MNK input and New Gamepad Driver are enabled.

### To Reproduce

Settings:
![Screenshot 2024-03-26 at 8 25 26 PM](https://github.com/unknownskl/greenlight/assets/39873075/ce131082-64cc-4c50-8a0c-7a5f772c969c)
I try to press a button once or move the mouse slightly, but sensitivity is ridiculously high and each key press registers at least twice.

### Expected behavior

Mouse and keyboard expected to input only once

### Environment

```markdown
- OS: MacOS Sonoma
- Device: MacBook Pro 13 inch Intel
- Greenlight version: 2.2.1
```

### New issue

- [X] I confirm that this is a new issue and not a duplicate

**Comment by LLuna-a** [comment]
Same on sequoia too
